packages/server/src/worker/processJobSyncAlgolia.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  JobKind,
+  TJobSyncAlgoliaData,
+  TJobSyncAlgoliaResult,
+} from "./workerTypes";
+import Resource, { resourceDocumentToResource } from "../models/Resource";
+
+import { Job } from "bullmq";
+import { TAlgoliaHit } from "@upswyng/types";
+import algoliaSearch from "algoliasearch";
+
+/**
+ * Update our Algolia search index with current resource data
+ */
+export async function processJobSyncAlgolia(
+  job: Job<TJobSyncAlgoliaData, TJobSyncAlgoliaResult>,
+  algoliaAppId: string,
+  algoliaIndexName: string,
+  algoliaWriteApiKey: string
+): Promise<TJobSyncAlgoliaResult> {
+  console.info(`${job.name}[${job.id}]\t: Syncing Resources to Algolia`);
+
+  // Initialize Algolia project and project index
+  const algolia = algoliaSearch(algoliaAppId, algoliaWriteApiKey);
+  const index = algolia.initIndex(algoliaIndexName);
+  console.info(
+    `${job.name}[${job.id}]\t: Algolia connection has been established`
+  );
+  job.updateProgress(25);
+
+  let wasUpdated = false;
+  wasUpdated = await Resource.getAll()
```

Comment:
This is my first time working with our database directly... so if `Resource.getAll()` was the wrong choice - let me know! Does this include the deleted resources? I could not really tell?